backend/usage/models.py

```diff
@@ -26,7 +26,18 @@ class Usage(BaseModel):
     run_id = models.CharField(
         max_length=255, null=True, blank=True, db_comment="Identifier for the run"
     )
-    usage_type = models.CharField(max_length=255, db_comment="Type of usage")
+    usage_type = models.CharField(
+        max_length=255, db_comment="Type of usage, either 'llm' or 'embedding'"
+    )
+    llm_usage_reason = models.CharField(
```

Comment:
@Deepak-Kesavan can this be `usage_reason` and in case of `embedding` this could further detail `indexing` / `querying` or any other purpose we choose to embed for

Reply:
https://github.com/Zipstack/unstract-sdk/pull/61#discussion_r1648826885

Resolving this